packages/d/dragselect.json

```
@@ -0,0 +1,43 @@
+{
+  "name": "dragselect",
+  "filename": "ds.min.js",
+  "description": "Easy JavaScript library for selecting and moving elements. With no dependencies. Drag-Select & Drag-And-Drop.",
+  "keywords": [
+    "javascript",
+    "lightweight",
+    "npm",
+    "drag-and-drop",
+    "drag",
+    "selection",
+    "production-ready",
+    "no-dependencies",
+    "dragselect",
+    "drag-n-drop",
+    "drag-selection",
+    "drag-select",
+    "drag-selection-library"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ThibaultJanBeyer/DragSelect.git"
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "npm",
+    "target": "dragselect",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
```

👋 We try to avoid using open globstars as it can lead to including files on the CDN that weren't intended. Looking at the package, a simple `*.js` would suffice here (if typings are needed, then an additional `**/*.d.ts` may be appropriate, though generally typings aren't needed on a web CDN).

```suggestion
          "*.js"
```

---

Thank you! Let's skip the typings for now then.